index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mulugeta-Portfolio</title>
+    <link rel="stylesheet" href="style/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- define container that holds all the items of the page-->
+    <div class="conatianer">
+
+        <!--Header part of the page-->
+        <header>
+            <p class="logo"> My Logo</p>
+            <nav class="hamburger">
+                <a href="#"> <img src="images/Union.png" alt="burgerIcon" class="burgerIcon"> </a>
+            </nav>
+        </header>
+        <!--Main part of the page-->
+        <main class="mainBody">
+            <section class="headLine">
+                <h1> I’m Mariane Glad to see you!  </h1>
+                <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
```

Comment:
- [ ] Nice design 👏 . Please make sure you align your content with `flexbox` to get used to it because you would be using it often as a developer. YOu can read more about `flexbox` in this [article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) 🙏 .